core/wasm/runtest/generate-wasm/to-wasm/src/lib.rs

```diff
@@ -39,8 +39,6 @@ extern "C" {
     fn return_value(value: *const u8);
     fn return_promise(promise_index: u32);
 
-    // Alias is standard length prefix buffer, but ID is always 32 bytes.
-    fn account_alias_to_id(account_alias: *const u8, account_id: *mut u8);
     // Sender's account id. Writes 32 bytes.
     fn sender_id(account_id: *mut u8);
```

Comment:
Please add a TODO here
I'll need to add sender_id_len function to get len of the account_id. Or refactor entire thing to get accounts by some enum, e.g. get_account_id, get_account_len which takes ACCOUNT_SENDER or ACCOUNT_SELF as first argument.